Gems/Atom/RHI/Vulkan/Code/Source/RHI/BufferView.cpp

```diff
@@ -80,30 +80,47 @@ namespace AZ
         }
 
         RHI::ResultCode BufferView::InvalidateInternal()
+        {
+            ReleaseView();
+            RHI::ResultCode initResult = InitInternal(GetDevice(), GetResource());
+            if (initResult != RHI::ResultCode::Success)
+            {
+                ReleaseBindlessIndices();
+            }
+            return initResult;
```

Comment:
Is it possible to make a similar change for metal too as this issue will exist on metal as well. I can help with testing.

Reply:
Sure - I'll look into metal, no way to compile that on my end though so it's a little tricky.